indico/htdocs/js/indico/RegistrationForm/tpls/fields/number.tpl.html

```diff
@@ -14,7 +14,7 @@
         {{ 'Field is mandatory' | i18n }}
     </span>
     <span ng-show="validationStarted && nestedForm.$invalid && nestedForm.$error.min " class="error-message-tag">
-        {{ 'The value must be at least' | i18n }} {{field.values.minValue}}
+        {{ 'The value must be at least' | i18n }} {{field.minValue}}
```

Comment:
That is not angular specific filter. It is implemented as a custom filter which uses `$T` function.

```

ndFilters.filter("i18n", function() {
    return function(input) {
        return $T(input);
    };
});

```